wedeploy/liferay/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mdelapenya/liferay-portal:7-ce-ga5-tomcat-hsql
+
+COPY portal-ext.properties $LIFERAY_HOME/portal-ext.properties
+
+ADD deploy $LIFERAY_HOME/deploy
+ADD osgi-configs $LIFERAY_HOME/osgi/configs
```

Comment:
Hey @lawrence-lee  we already have a mechanism for storing configs in liferay workspace (configs folder) can we re-use that somehow instead?